Classes/Core/Functional/Framework/DataHandling/Snapshot/DatabaseAccessor.php

```diff
@@ -146,6 +147,10 @@ private function importTable(string $tableName, array $columns, array $items)
                 );
             }
         }
+        // reset table sequences after inserting snapshot data. Dataset contains pk column data which
```

Comment:
What is "pk"?

Reply:
PK = primary key.

Reply:
Then please let's use the term "primary key" here.

Reply:
done